webapi/appsettings.json

```diff
@@ -13,13 +13,17 @@
   //
   // Service configuration
   // - Optionally set TimeoutLimitInS to the maximum number of seconds to wait for a response from the AI service. If this is not set, there is no timeout.
-  // - Optionally set SemanticSkillsDirectory to the directory from which to load semantic skills (e.g., "./SemanticSkills").
+  // - Optionally set:
+  //     - SemanticSkillsDirectory to the directory from which to load semantic skills (e.g., "./SemanticSkills").
+  //     - NativeSkillsDirectory to the directory from which to load native skills (e.g., "./NativeSkills").
+  //     - Note: Make sure to have corresponding paths in CopilotChatWebApi.csproj to copy these skills to the output directory.
```

Comment:
Is this still true??

Reply:
No, can you see the updated README / comment?